Add two-fer disapproval comments (#1244)
exercism/website-copy: automated-comments/ruby/general/explicit_return.md

Remove the explict return to simplify the method.

> In Ruby, a method will return the object that was returned from the last evaluated statement. Explicitly using the `return` keyword is therefore unnecessary unless we are returning early in a method.

Reviewer: ```suggestion
In Ruby, a method will return the result of the last evaluated statement. Explicitly using the `return` keyword is therefore unnecessary, unless when returning early in a method.
```

A bit more concise and without personas?

Author: Updated `unless` for `except`

> For example, this will print `16` as the `multiply` method will return the result of `a * b`, it's final expression.

Reviewer: ```suggestion
For example, this will print `16` as the `multiply` method returns the result of `a * b`, its final expression.
```

I think that's meant to be just possessive?

Author: Ridiculous language ;)
